docs/layernorm.rst

```diff
@@ -0,0 +1,17 @@
+
+Layernorm Layer
+===================
+
+The layernorm  types and functions.
```

Comment:
@seungmanhan Please mention that this is experimental functionality and the user need to define MIOPEN_BETA_API before inclusion of miopen.h to enable it.